### DISCUSSION TITLE
aiuto con colonna compiti

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#669] [Richieste di aiuto]
Buongiorno allego l'immagine dove ho creato semplicemente un'altra colonna dedicata ai compiti ,solo che non ho capito il perché le icone "aggiungi" che normalmente si usano per inserire la lezione siano finite all'interno della stessa colonna dei compiti.Vi spiego i passaggi che ho fatto : templates>registro_firme_G.html.twig e da qui nella seguente parte come si può vedere ho ricopiato la stessa struttura dell'argomento rinominandola con il nome "Compiti".
     " <th class="col-sm-2">Ora</th>
        <th class="col-sm-2">Materia</th>
        <th class="col-sm-2">Docenti</th>
        <th class="col-sm-4">Argomenti/Attività</th>
        <th class="col-sm-2">Compiti</th>
        <th class="col-sm-2">Azioni</th>
 Da qui poi mi sono recato in PhpMyAdmin ,dove ho eseguito i seguenti passi : gs_lezione>new> e da qui ho creato la colonna intitolata "compiti" , ma a quanto pare questo è quello che mi esce (allego foto ) e tra l'altro ,quando mi reco sul tasto aggiungi ,non vedo la cella compiti , ma solo quella per l'argomento e l'attività. Mi potreste per favore dirmi nel modo più semplice possibile come risolvere il problema con un'eventuale giusta correzione ?Grazie mille

<img width="1422" alt="Screenshot 2024-03-17 alle 22 16 22" src="https://github.com/iisgiua/giuaschool/assets/93823125/2e8fa454-5656-4e04-a97a-102c47b4103f">

**Comment by trinko** [comment]
Sicuramente viene visualizzato così perché hai aggiunto la colonna nell'intestazione della tabella ma non nelle righe del contenuto, cioè all'interno dei tag 
```<tbody>...<tr><td> dato...</td> <td> dato...</td> ...```

Per il resto, quello che vuoi ottenere non è così semplice... 
- devi aggiungere il nuovo dato all'entità Lezione (file **src/Entity/Lezione.php**)
- devi aggiungere il nuovo dato nel form nelle funzioni **add(..)** e **edit(..)** (file **src/Controller/Registro.php**)
- devi modificare i vari template in cui vengono mostrati i contenuti delle lezioni ai docenti/genitori/alunni

**Comment by trinko** [comment]
Comunque continuo a non capire perché i compiti non possano essere scritti nel ampo "Attività".
